### PR TITLE
Introduction section for Maui/Mahuika

### PR DESCRIPTION
**Pull request by tinyendian**
A first shot at this section, including architecture diagrams. Someone from the platforms team should have a look at this and check if the diagrams and technical data are accurate.

I thought it best to split the diagrams in two, to make it a bit easier to understand. Most users will only access one of the two systems. I thus included the storage system twice, on the Maui and Mahuika diagrams, and explained in the text that there is only one shared system.

### CODE REVIEW

File: _lessons/maui-and-mahuika/01-introduction.md [hunk 1]
```diff
@@ -13,4 +13,98 @@ You will learn:
...
+* Run on a full version of SLES
+
+#### Storage
+* Dedicated cluster for the Shared Service nodes
+* InfiniBand network for access from build nodes and CSx00 nodes to ensure high throughput and low latency
+* Cray DVS provides access to Spectrum Scale from compute nodes via the Aries interconnect
+
+#### Main improvements
+* Faster processors compared to those on FitzRoy or Pan
+* GPGPU (General Purpose Graphical Processing Unit) nodes to support science codes and visualisation
```
Comment (chrisdjscott) [inline _lessons/maui-and-mahuika/01-introduction.md]: Is it worth mentioning that the GPUs are much more powerful than the ones on Pan (that's a big improvement)?

File: _lessons/maui-and-mahuika/01-introduction.md [hunk 2]
```diff
@@ -13,4 +13,98 @@ You will learn:
...
+* A Linux user environment that will make it easier to manage work, develop and run research workloads/jobs, and apply data analytics tools
```
Comment (chrisdjscott) [inline _lessons/maui-and-mahuika/01-introduction.md]: Maybe add "(compared to Fitzroy)" at the end as Pan was running Linux already?

**Comment by chrisdjscott** [comment]
Thanks @tinyendian this is great. Couple of comments but looks good to me

**Comment by tinyendian** [comment]
Thanks @chrisdjscott, good points - I applied the suggested changes (and fixed another typo) and will merge the pull request.

**Pull request merged**